hiera.yaml

```diff
@@ -12,6 +12,9 @@ hierarchy:
   - name: 'Major Version'
     path: '%{facts.os.name}/%{facts.os.release.major}.yaml'
 
+  - name: 'Family - Major Version'
+    path: '%{facts.os.family}-family-%{facts.os.release.major}.yaml'
+
```

Comment:
I do not like  that, since the release number is name and not family specific (at least in Debian/Ubuntu). So I think we  should use the already existing name specific entry.
Also means move RedHat-familiy-9.yaml to RedHat/9.yaml and Centos/9.yaml (no idea if CentOS 9 is still binary compatible with RedHat...).